fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSitesViewHolder.kt

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.topsites
+
+import android.view.View
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import androidx.lifecycle.LifecycleOwner
+import mozilla.components.feature.top.sites.TopSite
+import mozilla.components.lib.state.ext.observeAsComposableState
+import org.mozilla.fenix.components.components
+import org.mozilla.fenix.compose.ComposeViewHolder
+import org.mozilla.fenix.home.sessioncontrol.TopSiteInteractor
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * TODO
+ */
+class TopSitesViewHolder(
+    composeView: ComposeView,
+    viewLifecycleOwner: LifecycleOwner,
+    private val interactor: TopSiteInteractor,
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
+
+    @Composable
+    override fun Content() {
+        composeView.setContent {
+            val topSites =
+                components.appStore.observeAsComposableState { state -> state.topSites }.value
+                    ?: emptyList()
+
+            FirefoxTheme {
```

Comment:
don't need the firefoxtheme wrapper here, as that happens within ComposeViewHolder

Reply:
Removed